proposal.html

```diff
@@ -485,7 +485,7 @@
 
   <ins class="block">
     <emu-alg>
-      1. If _func_ is a <emu-xref href="#sec-bound-function-exotic-objects">Bound Function exotic object</emu-xref> or a <emu-xref href="#sec-ecmascript-standard-built-in-objects">built-in Function object</emu-xref>, then return an implementation-dependent String source code representation of _func_. The representation must have the syntax of a |NativeFunction|.
+      1. If _func_ is a <emu-xref href="#sec-bound-function-exotic-objects">Bound Function exotic object</emu-xref> or a <emu-xref href="#sec-ecmascript-standard-built-in-objects">built-in Function object</emu-xref>, then return an implementation-dependent String source code representation of _func_. The representation must have the syntax of a |NativeFunction|. Additionally, if _func_ is a <emu-xref href="#sec-well-known-intrinsic-objects">Well-known Intrinsic Object</emu-xref> and the Global Name associated with _func_ has the syntax of a |BindingIdentifier|, the portion of the returned String that would be matched by |BindingIdentifier| must be the same as the Global Name associated with _func_.
```

Comment:
An excerpt from [the notes](https://github.com/tc39/tc39-notes/blob/924122cdc03e9ee2afbe8014193f845bddc6da2d/es7/2016-01/2016-01-28.md#functiontostring--functionisportable-mf):

> WH: What case does calling toString on Math.sin fall into?

> MF: The last one: [native code].

> AWB: What if I want Math.sin to be self-hosted in ECMAScript?

> MF: Host can control the internal slot...

> AWB: Not if it's an ecmascript function object. An ECMAScript function object is an object that is created syntactically from ECMAScript code.

> MF: implementation cna pretend there's a slot.

> AWB: Concern is for self hosted built-ins you want them to be opaque.

> MF: We can add a case for built-in functions. Is that something that we want to do?

> BT: We should mandate that built-in functions show [Native Code]

> YK/MF: Agree.

So @bterlson wanted implementations to be prevented from pretending that a built-in was defined using ECMAScript code.